vignettes/methods_comparison/methods_comparison.Rmd

```diff
@@ -23,14 +26,18 @@ Our dataset therefore offers the ideal platform for techincal comparisons of seq
 
 <!--We found that the number of detected genes decreases monotonically with increasing thresholds at similar rates across different organs and tissues within each method. We observed that in the droplet data more than half of the detected genes are represented by only a single UMI; this is to be expected given that only a few thousand UMIs are captured per cell. The FACS data are sampled much more deeply and one needs to set a relatively high threshold of 40 reads to see a comparable reduction in gene detection sensitivity.-->
 
-In parallel with our mouse cell atlas, [Han et al](http://www.cell.com/cell/abstract/S0092-8674(18)30116-8) developed a new method, microwell-seq, and also derived another mouse cell atlas. It is pertinent to compare these two works because together they offer a more complete picture of mouse biology at single cell resolution. We decided to compare the data at the level of differentially expressed genes. To do so, we computed [lists of genes](https://github.com/czbiohub/tabula-muris/blob/master/22_markers/FindMarkers.Rmd) that differentiate each cell type from the rest of the cells for a given organ across all three methods, focusing on common organs and cell types for the three methods. The following code snippets are from the Matlab scripts available [here](https://github.com/czbiohub/tabula-muris/blob/master/22_markers/methods_comparison.m) and for simplicity  we also provide the [gene signature files](https://github.com/czbiohub/tabula-muris/tree/master/22_markers) for each organ. There are 8 tissues for which we have data from the three methods:
+In parallel, [Han et al](http://www.cell.com/cell/abstract/S0092-8674(18)30116-8) developed a new method, microwell-seq, and also derived another mouse cell atlas. It is pertinent to compare these two works because together they offer a more complete picture of mouse biology at single cell resolution.
```

Comment:
I wouldn't say Han et al developed microwell-seq–Guoji Guo has used it in the past. Could just rephrase to say "using microwell seq" and cite https://www.biorxiv.org/content/early/2017/04/13/127217